websites/K/Kick/presence.ts

```diff
@@ -0,0 +1,65 @@
+const presence = new Presence({ clientId: "1112901248421732462" });
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/kVabJjN.png",
+			startTimestamp: Date.now(),
+		},
+		pathDetailsMap = {
+			"/terms-of-service": {
+				details: "Reading",
+				state: "Terms of Service",
+				largeImageKey: "",
+			},
+			"/privacy-policy": {
+				details: "Reading",
+				state: "Privacy Policy",
+				largeImageKey: "",
+			},
+			"/dmca-policy": {
+				details: "Reading",
+				state: "DMCA Policy",
+				largeImageKey: "",
+			},
```

Comment:
Remove unused properties
```suggestion
			"/terms-of-service": {
				details: "Reading",
				state: "Terms of Service",
			},
			"/privacy-policy": {
				details: "Reading",
				state: "Privacy Policy",
			},
			"/dmca-policy": {
				details: "Reading",
				state: "DMCA Policy",
			},
```